lib/controllers/ephemeral/account-linking/available_fsp_controller.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:async';
+
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:get/get.dart';
+import 'package:pispapp/models/fsp.dart';
+import 'package:pispapp/repositories/firebase/participant_repository.dart';
+
+class AvailableFSPController extends GetxController {
+  AvailableFSPController(this._participantRepository);
+
+  final ParticipantRepository _participantRepository;
+  RxList<Fsp> availableFsps = <Fsp>[].obs;
+
+
+  @override
+  void onInit() {
+    availableFsps.bindStream(_participantRepository.listenForAvailableFSPs());
```

Comment:
I don't think we need to bind the stream and keep listening here...
Why don't we just perform a get query when opening the screen?

Reply:
Yeah I guess that makes sense - I initially wrote it to just fetch once lol but then it seemed better at the time but I guess its unnecessary to listen